### PR TITLE
bug fix: SLR CIS map layer redraw

### PR DESCRIPTION
**Pull request by clhenrick**
Fixes a couple bugs relating to the Map data layers:
- when navigating to a new route/page in the website from the SLR CIS tool the requested page would not load
- when navigating back to the SLR CIS tool from another page the map layers would not draw

### CODE REVIEW

**Comment by clhenrick** [comment]
[Bug: On DEV/BETA SLR-CIS tool has top menu issue](https://trello.com/c/7DhLPpSX/187-bug-on-dev-beta-slr-cis-tool-has-top-menu-issue)

File: src/routes/tools/slr-coastal-inundation/_Map/Rasters.svelte [hunk 1]
```diff
@@ -104,10 +108,10 @@
   }
 
   onDestroy(() => {
-    map.off("styledata", handleStyleDataChange);
     removeRasterLayers();
     removePreviousRasterLayers();
     layerHandler.removeMapRef();
     layerHandler = null;
+    map.off("styledata", handleStyleDataChange);
```
Comment (elehmer) [inline src/routes/tools/slr-coastal-inundation/_Map/Rasters.svelte]: Do you want to move this map.off to the bottom of onDestroy in TileIndexes and TileCentroids as well?

**Review by elehmer — COMMENTED** [review]
LGTM but wondering if you want to move map.off operation in onDestroy to the bottom of TileIndexes and TileCentroids as well.

File: src/routes/tools/slr-coastal-inundation/_Map/Rasters.svelte (hunk 1 above)
Comment (clhenrick) [reply src/routes/tools/slr-coastal-inundation/_Map/Rasters.svelte]: It probably doesn't matter either way but might be good to be consistent.

Comment (elehmer) [reply src/routes/tools/slr-coastal-inundation/_Map/Rasters.svelte]: Yeah agreed but get style points!

**Pull request merged**